Building with make devweb Fails.

When building the project on local machine with ```make devweb``` command fails with following error. 
```
Building devweb
Step 1/17 : FROM channelapi_uwsgi
ERROR: Service 'devweb' failed to build: repository channelapi_uwsgi not found: does not exist or no pull access
make: *** [devweb] Error 1
```
Investigate and correct the problem.

--- a/deployment/docker/Dockerfile-dev
+++ b/deployment/docker/Dockerfile-dev
@@ -1,6 +1,5 @@
 #--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
 # Note this base image is based on debian
-# FROM channelapi_uwsgi
 FROM channelapi_web
 MAINTAINER Tim Sutton<tim@kartoza.com>
 
@@ -28,7 +27,6 @@ RUN echo "export VISIBLE=now" >> /etc/profile
 
 ADD REQUIREMENTS-dev.txt /REQUIREMENTS-dev.txt
 RUN pip install -r /REQUIREMENTS-dev.txt
-# RUN pip uninstall -y South
 ADD bashrc /root/.bashrc
 
 # --------------------------------------------------------

--- a/deployment/docker/Dockerfile-dev
+++ b/deployment/docker/Dockerfile-dev
@@ -1,6 +1,7 @@
 #--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
 # Note this base image is based on debian
-FROM channelapi_uwsgi
+# FROM channelapi_uwsgi
+FROM channelapi_web
 MAINTAINER Tim Sutton<tim@kartoza.com>
 
 # Use local cached debs from host (saves your bandwidth!)
@@ -27,7 +28,7 @@ RUN echo "export VISIBLE=now" >> /etc/profile
 
 ADD REQUIREMENTS-dev.txt /REQUIREMENTS-dev.txt
 RUN pip install -r /REQUIREMENTS-dev.txt
-RUN pip uninstall -y South
+# RUN pip uninstall -y South
 ADD bashrc /root/.bashrc
 
 # --------------------------------------------------------

